code/modules/projectiles/projectile/bullets.dm

```diff
@@ -115,16 +115,17 @@
 	weaken = 0
 
 /obj/item/projectile/bullet/fourtyfive //secgun ammo!
-	damage = 15
-	stun = 2 //stopping power
-	weaken = 2
+	damage = 13
+	drowsy = 0.25 //stopping power
+	agony = 0.25
 	penetration = 2
 
 /obj/item/projectile/bullet/fourtyfive/practice
 	damage = 2
-	stun = 1
-	weaken = 1
+	drowsy = 0
+	agony = 0
 	embed = 0
+	penetration = 0
```

Comment:
Nevermind, I see that you're talking about a child of `/obj/item/projectile/bullet/fourtyfive`.